package.json

```diff
@@ -955,6 +964,11 @@
           "when": "view =~ /prStatus/",
           "group": "navigation"
         },
+        {
+          "command": "pr.configurePRViewlet",
```

Comment:
I think we could combine this with the entry above for configurePRViewlet by changing the when clause there